rfd/0089-merge-webapps.md

```diff
@@ -195,7 +195,7 @@ build systems to successfully build Teleport.
       the correct version in the `teleport.e/web` folder.
 - [ ] Only require teleport build processes to run on teleport paths and the webapp
       ones to run on the webapp paths
-- [ ] Update Connect's build pipelines (in teleport/dronegen) as webapps will no longer need to be cloned.
```

Comment:
I'd typically refrain from messing with RFDs, as these are historical artifacts.  However this was the only use of dronegen remaining in the repo, and it allowed us to clean up the cspell.json configs.  I figured it wasn't adding anything meaningful to the RFD and was safe to clean up.

If you feel otherwise, let me know and I'll drop this and the corresponding cspell cleanups.